command/quota_status.go

```diff
@@ -26,13 +26,26 @@ Usage: nomad quota status [options] <quota>
 
 General Options:
 
-  ` + generalOptionsUsage(usageOptsDefault)
+  ` + generalOptionsUsage(usageOptsDefault) + `
+
+  Status Specific Options:
+	
+	-json
+	  Output the latest quota status information in a JSON format.
+	
+	-t
+	  Format and display quota status information using a Go template.
+  `
```

Comment:
slight indentation change:
```suggestion
Status Specific Options:
	
  -json
    Output the latest quota status information in a JSON format.
	
  -t
    Format and display quota status information using a Go template.
`
```